tokio-trace-fmt/src/filter.rs

```diff
@@ -256,9 +263,28 @@ impl Directive {
             }
         });
 
-        let in_span = caps
+        let (in_span, fields) = caps
             .name("span")
-            .map(|c| c.as_str().trim_matches(|c| c == '[' || c == ']').to_owned());
+            .and_then(|cap| {
+                let cap = cap.as_str().trim_matches(|c| c == '[' || c == ']');
+                let caps = SPAN_PART_RE.captures(cap)?;
+                let span = caps.name("name").map(|c| c.as_str().to_owned());
+                println!("SPAN PART IS {:?}", span);
```

Comment:
sneaky print

Reply:
whoops, i meant to get rid of that, thanks!